charts/operator/values.yaml

```
@@ -33,4 +33,8 @@ clusterRole:
       resources: ["*"]
       verbs: ["*"]
 
+httpProxy:
+  name: kube-rbac-proxy
+  image: gcr.io/kubebuilder/kube-rbac-proxy:v0.5.0
```

```suggestion
  image:
    repository: gcr.io/kubebuilder/kube-rbac-proxy
    tag: v0.5.0
```

We dont need to template name